Wireframe/index.html

```diff
@@ -9,25 +9,55 @@
   <body>
     <header>
       <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
-      </p>
+      <p> What Is a Wireframe? </p>
     </header>
     <main>
       <article>
-        <img src="placeholder.svg" alt="" />
-        <h2>Title</h2>
+        <img
+          src="https://balsamiq.com/assets/learn/articles/account-setup-wireframe.png"
+          alt=""
```

Comment:
Can you review this page for information about the `alt` attribute? https://www.w3schools.com/tags/att_img_alt.asp